cyber/common/file.cc

```diff
@@ -122,11 +122,12 @@ bool GetContent(const std::string &file_name, std::string *content) {
   return true;
 }
 
-std::string GetAbsolutePath(const std::string &prefix,
+const std::string GetAbsolutePath(const std::string &prefix,
```

Comment:
Actually there is no benefit to apply "const" at the util side which is simply a *copy* of string.
Caller could choose to get a const or non-const result based on their need.

Reply:
For example, when we add "const" here, user could still call it with
```C++
string path = GetAbsolutePath();
```

which makes things worse because the string has to be copied once. While without const here, user always gets the efficiency of RVO, no matter it's kept as const or non-const.

About RVO, please check this page: https://abseil.io/tips/11

Reply:
I was not aware of that, Thanks Xiangquan for pointing this out!